### PR TITLE
Make hover integration test pass

### PR DESCRIPTION
**Pull request by akshita31**
The hover integration test is failing because of a possible race condition between the omnisharp process and the vscode process to access the file that has been added. Hence, if we restart the server before we ask vscode to open it, omnisharp will have read the file and no problem should occur.

PS: I am not sure whether this is the best approach to solve this, because we will have to restart the server whenever we add a file for testing.

### CODE REVIEW

**Comment by rchande** [comment]
@akshita31 The solution we discussed was to just always include the "hover" file in the test solution. However, if this works, I'm happy to take it :). I restarted your PR build to see if it's flaky.

File: test/integrationTests/hoverProvider.integration.test.ts [hunk 1]
```diff
@@ -48,15 +48,11 @@ namespace Test
...
 }`;
-        let fileUri = await testAssetWorkspace.projects[0].addFileWithContents("test1.cs", program);
-
+        let fileUri = await testAssetWorkspace.projects[0].addFileWithContents("hover.cs", program);
         await omnisharp.restart();
         await omnisharp.waitForEmptyEventQueue();
-
-        await vscode.workspace.openTextDocument(fileUri);
-
+        await vscode.commands.executeCommand("vscode.open", fileUri);
```
Comment (rchande) [inline test/integrationTests/hoverProvider.integration.test.ts]: Why did you move away from the openTextDocument helper that you switched to?

Comment (akshita31) [reply test/integrationTests/hoverProvider.integration.test.ts]: No specific reason. I just thought it would maintain consistency with the codeActionRename test. Is openTextDocument a better way ?

**Comment by akshita31** [comment]
@rchande I was trying to add a test for signature help for this PR : https://github.com/OmniSharp/omnisharp-vscode/pull/1958 , following the same approach as above, but it doesn't seem to work :(

**Comment by akshita31** [comment]
This approach doesn't work. Closing this in favor of https://github.com/OmniSharp/omnisharp-vscode/pull/2058
